### ISSUE TITLE
Mac OS X 10.9 - Cannot name project when building .shy

### ISSUE THREAD

**Issue by tomkadwill** [shoes/shoes-deprecated#269]
When using the OS X app to 'Package my script (shy)'  I cannot enter a Project Name, Version or Your Name. When I enter text and move to the next field, my test disappears.

Let me know if you need any more detail :)

**Comment by ccoupe** [comment]
Thats odd.  Which version of Shoes are you using? If it's 3.2 you should have a Maintain Shoe link on the splash screen and from there  you can get some build info. I just checked my OSX system and the build on 20140903 doesn't exhibit the problem you describe.

**Comment by tomkadwill** [comment]
Yes I am using 3.2, I've tested on two different machines (both OSX 10.9) and have the same problem on both. I'm also using 20140903.

I can actually create a package, I just can't give it a name, author, or version. It just assigns a default name `shoes-3.2.15-osx-10.9.tgz`

**Comment by ccoupe** [comment]
Very odd. That's Ruby code that hasn't been changed in ages. (Shoes.app/Contents/MacOS/lib/shy*.rb). One speculation is that perhaps Shoes doesn't like the UK keycodes. That sounds like crazy talk to me. Can you try some of the Samples from the manual (AndSoForth section) and enter text?  Simple-controls should work.

**Comment by ccoupe** [comment]
Tom, are you tabbing to the next field in the Shy screen or clicking to the next field. Tab navigation (focus) is a problem on OSX (all Shoes actually but worse on OSX)

**Comment by tomkadwill** [comment]
@ccoupe I've tried tab navigation and clicking on the next field. Both methods result in the same bug. It is strange that it has broken when the code hasn't changed. Perhaps the latest version of OSX has broken it?

I've also noticed that Shoes is quite un-stable, its crashes and then has trouble starting up again. Perhaps this is also due to the latest OSX upgrade.

**Comment by ccoupe** [comment]
Tom,  I'm seeing some weirdness too with the 3.2.15 download now (doesn't work at all).  Give 3.2.14 a try at http://shoes.mvmanila.com/public/shoes/  That seems to work better.

**Comment by ccoupe** [comment]
Or the 3.2.16 beta in http://shoes.mvmanila.com/public/shoes/beta/ - don't the packaging features with
this one.

**Comment by tomkadwill** [comment]
Cool thanks. 3.2.14 worked for me but not 3.2.16 beta (just so you know). Thanks for the help! :)

**Comment by ccoupe** [comment]
Tom,  I've taken another look at this issues.  The latest Shoes 3.2.16 beta works ok for me. Do you have other versions of Shoes on your box(en)?

**Comment by SteamComputing** [comment]
3.2.16 seems to be broken on this.

```
Shoes.app do
  edit_line
  edit_line
end
```

If I enter something in the first field, then move to the 2nd. The string in the first field disappears. But the 3.2.15 build on Oct 24 seems to be fine. My OS is 10.10

**Comment by ccoupe** [comment]
@SteamComputing. Thanks for reporting  and including an easy test case and work around.  Thanks. Definitely a problem.

**Comment by ccoupe** [comment]
I found the problem. It will be fixed in 3.2.17. If you promise not to package things for Windows, you can get a beta with the fix from http://walkabout.mvmanila.com/public/shoes/shoes-3.2.17-osx-10.9.tgz
